### PR TITLE
feat(draco): writer worker

### PR DESCRIPTION
**Pull request by belom88**
I created 2 workers: for NodeJS and for Browser. The worker for a web browser doesn't work because of `importScripts`-related issue.

### CODE REVIEW

File: modules/draco/package.json [hunk 1]
```diff
@@ -34,13 +34,17 @@
   "scripts": {
     "pre-build": "npm run build-bundle && npm run build-worker",
     "build-bundle": "esbuild src/bundle.ts --outfile=dist/dist.min.js --bundle --minify --sourcemap",
-    "build-worker": "esbuild src/workers/draco-worker.ts --outfile=dist/draco-worker.js --bundle --minify --sourcemap --define:__VERSION__=\\\"$npm_package_version\\\""
+    "build-loader-worker": "esbuild src/workers/draco-worker.ts --outfile=dist/draco-worker.js --bundle --minify --sourcemap --define:__VERSION__=\\\"$npm_package_version\\\"",
+    "build-writer-worker": "esbuild src/workers/draco-writer-worker.ts --outfile=dist/draco-writer-worker.js --minify --bundle --sourcemap --define:__VERSION__=\\\"$npm_package_version\\\"",
+    "build-writer-nodejs-worker": "esbuild src/workers/draco-writer-nodejs-worker.ts --outfile=dist/draco-writer-nodejs-worker.js --platform=node --target=esnext,node12 --minify --bundle --sourcemap --define:__VERSION__=\\\"$npm_package_version\\\"",
```
Comment (ibgreen) [inline modules/draco/package.json:39]:
```suggestion
    "build-writer-worker-nodejs": "esbuild src/workers/draco-writer-nodejs-worker.ts --outfile=dist/draco-writer-nodejs-worker.js --platform=node --target=esnext,node12 --minify --bundle --sourcemap --define:__VERSION__=\\\"$npm_package_version\\\"",
```

File: modules/draco/package.json [hunk 2]
```diff
@@ -34,13 +34,17 @@
...
+    "build-worker": "yarn build-loader-worker && yarn build-writer-worker && yarn build-writer-nodejs-worker"
   },
   "dependencies": {
     "@babel/runtime": "^7.3.1",
     "@loaders.gl/loader-utils": "4.0.0-alpha.5",
     "@loaders.gl/schema": "4.0.0-alpha.5",
     "@loaders.gl/worker-utils": "4.0.0-alpha.5",
+    "@loaders.gl/polyfills": "4.0.0-alpha.5",
```
Comment (ibgreen) [inline modules/draco/package.json:47]: I see, I wish we could make this a devDependency... the design was that the user chooses to install this. 

What exactly in polyfills are we using?

File: modules/draco/src/index.ts [hunk 3]
```diff
@@ -14,6 +15,30 @@ export type {DracoMesh, DracoLoaderData};
 export type {DracoWriterOptions} from './draco-writer';
 export {DracoWriter} from './draco-writer';
 
+/**
+ * Browser worker doesn't work because of issue during "draco_encoder.js" loading.
+ * Refused to execute script from 'https://raw.githubusercontent.com/google/draco/1.4.1/javascript/draco_encoder.js' because its MIME type ('') is not executable.
+ */
+export const DracoWriterWorker = {
+  id: 'draco-writer',
```
Comment (ibgreen) [inline modules/draco/src/index.ts:23]: Can we get away with only defining one DracoWriter, and hide the platform dependent parts inside:

```suggestion
  id: isBrowser ? 'draco-writer' : 'draco-writer-nodejs`,
```

**Review by ibgreen — APPROVED** [review]

File: modules/draco/package.json (hunk 2 above)
Comment (belom88) [reply modules/draco/package.json:47]: fetch api

File: modules/draco/package.json (hunk 1 above)
Comment (belom88) [reply modules/draco/package.json:39]: Done

File: modules/draco/src/index.ts (hunk 3 above)
Comment (belom88) [reply modules/draco/src/index.ts:23]: Done

**Pull request merged**